hilda/snippets/mach/CFRunLoopServiceMachPort_hooks.py

```diff
@@ -1,45 +1,41 @@
 import lldb
 
-from hilda.exceptions import SymbolAbsentError
 
-
-def _CFRunLoopServiceMachPort_hook(hilda, *args):
-    """
-    :param hilda.hilda_client.HildaClient hilda:
+def disable_mach_msg_errors():
     """
-    hilda.jump(hilda.CFRunLoopServiceMachPort_while_ea)
-    hilda.cont()
+    Disable mach_msg syscall timeout CFRunLoopServiceMachPort. `__CFRunLoopServiceMachPort` receives
```

Comment:
Remove the timeout validation from `__CFRunLoopServiceMachPort`.

This is done by patching the `mach_msg_timeout_t` parameter (4rd one) to `MACH_MSG_TIMEOUT_NONE`.